Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -26,36 +27,35 @@
 #include "DolphinQt/QtUtils/SetWindowDecorations.h"
 #include "DolphinQt/Settings.h"
 
+#include "Common/CommonTypes.h"
 #include "VideoCommon/PostProcessing.h"
 #include "VideoCommon/VideoBackendBase.h"
 #include "VideoCommon/VideoCommon.h"
 #include "VideoCommon/VideoConfig.h"
 
-EnhancementsWidget::EnhancementsWidget(GraphicsWindow* parent) : m_block_save(false)
+EnhancementsWidget::EnhancementsWidget(GraphicsWindow* parent)
 {
   CreateWidgets();
   LoadSettings();
   ConnectWidgets();
   AddDescriptions();
-  connect(parent, &GraphicsWindow::BackendChanged,
-          [this](const QString& backend) { LoadSettings(); });
-  connect(parent, &GraphicsWindow::UseFastTextureSamplingChanged, this,
-          &EnhancementsWidget::LoadSettings);
-  connect(parent, &GraphicsWindow::UseGPUTextureDecodingChanged, this,
```

Comment:
Just trying to understand the reason these were removed.  I believe these were removed because you added a generic `ConfigChanged`.  Is that correct?  And `BackendChanged` is still needed because there are times the backend changes but config doesn't?

Reply:
Yeah, ConfigChanged covers most things.  BackendChanged could probably be put into the ConfigChanged system, but keeping it separate could help organize those specific settings. GeneralWidget::OnEmulationStateChanged  will check if the backend changed on startup and emit the signal if it did.

Thinking about it, keeping a list of what is interconnected would be good.